gapic/samplegen/samplegen.py

```diff
@@ -297,7 +324,49 @@ def validate_response(self, response):
 
             validater(self, body)
 
-    def _handle_lvalue(self, lval):
+    def validate_expression(self, exp: str) -> wrappers.Field:
```

Comment:
question: why doesn't this method start with `_`? It does not seem to be used outside this class.

Reply:
I wanted to call it directly in the unit tests, which from one point of view promotes it into a first class portion of the class's API.